flan/src/lib/FileInput.svelte

```diff
@@ -46,6 +45,31 @@
 						Remove image
 					</button>
 				</div>
+			{:else if inputFile.type.includes('audio/')}
+				<div
+					class="grid grid-rows-[1fr_auto] gap-4 bg-secondary text-white p-4 rounded-lg">
+					<!-- class="grid grid-rows-[1fr_auto] gap-4" -->
+					<code class="break-all">{inputFile.name}</code>
```

Comment:
This code block seems to also be the same between all, image doesnt have it though unfortunately so moving outside if block seems good

Reply:
addressing this one by moving div outside if scope